colossus-docs/src/main/scala/PlayingWithServiceClientCallbacks.scala

```diff
@@ -0,0 +1,49 @@
+import java.net.{InetAddress, InetSocketAddress}
+
+import akka.actor.ActorSystem
+import akka.util.ByteString
+import colossus.IOSystem
+import colossus.metrics.logging.ColossusLogging
+import colossus.protocols.http.HttpMethod._
+import colossus.protocols.http.UrlParsing._
+import colossus.protocols.http.{Http, HttpServer, Initializer, RequestHandler}
+import colossus.protocols.redis.Redis
+import colossus.service.GenRequestHandler.PartialHandler
+
+import scala.util.{Failure, Success, Try}
+
+object PlayingWithServiceClientCallbacks extends App with ColossusLogging {
+
+  implicit val actorSystem = ActorSystem()
+  implicit val ioSystem    = IOSystem()
+  implicit val ec          = actorSystem.dispatcher
+
+  HttpServer.start("example-server", 9000) { initContext =>
+    new Initializer(initContext) {
+
+      val hosts = Seq(
+        new InetSocketAddress("localhost", 6379),
+        new InetSocketAddress("localhost", 6379),
+        new InetSocketAddress("localhost", 6379),
+        new InetSocketAddress("localhost", 6379),
+        new InetSocketAddress("localhost", 6379)
+      )
+
+      // if we need to change the hosts, then we would create a new instance
```

Comment:
i.e. no need to add/remove clients. Instead disconnect the old redis client and create a new redis client.

Reply:
To be able to update the client/load balancer we would need to have an `update` method on the `Sender` trait, which is fine, but I suppose I was trying to avoid touching it. Your point about connection churn makes a lot of sense.